src/api/tunnels.rs

```diff
@@ -44,6 +51,14 @@ pub struct CreateCommand {
     /// The length of time in seconds for the tunnel to live.
     #[arg(long)]
     ttl: Option<u16>,
+
+    /// Whether or not to wait for a state other than "requested"
+    #[arg(long)]
+    wait: Option<bool>,
+
+    /// The number of attempts to be used with "--wait". Max: 5, Min: 1
+    #[arg(long)]
+    attempts: Option<u8>,
```

Comment:
@jasontwong lets adjust this. Remove the `attempts` arg and change the `wait` flag semantics so that it takes a number of seconds to wait for. So as a user I can just `--wait 30` if I was willing to wait 30 seconds.

Max wait is 3600 seconds. I think you can specify this alongside where you see arg(long) so that clap both enforces the min max for you and auto appends it to your docs so you wouldn't have to manually write that either.

The sleep duration (assuming 0 based index in the for loop where `x` is the iteration and that the sleep happens after the attempt in the loop body) should be `(x + 2) ^ 2`. So the sleeps would be: 4, 9, 16, 25, 36....

You will need to break before sleeping again if "now - start" is > provided-wait-time.